transports/webrtc/src/tokio/udp_mux.rs

```diff
@@ -398,11 +398,19 @@ impl UDPMuxNewAddr {
 
                             continue;
                         }
-                        Poll::Ready(Err(err)) if err.kind() == ErrorKind::TimedOut => {}
                         Poll::Pending => {}
                         Poll::Ready(Err(err)) => {
-                            log::error!("Could not read udp packet: {}", err);
-                            return Poll::Ready(UDPMuxEvent::Error(err));
+                            match err.kind() {
+                                ErrorKind::TimedOut => {}
+                                // If client drops off and resets the connection, do nothing with the ListenStream
+                                ErrorKind::ConnectionReset => {
+                                    log::debug!("ConnectionReset by remote client {err:?}");
```

Comment:
Can we include any connection details here?

Reply:
I've been looking, but I still don't see how we could correlate this `ConnectionReset` error to any given `connection`.

The operating system error (code 10054) gives us no information about the connection or how to identify which connection (it actually says "**_An_** existing connection was forcibly closed" -- but not which one). 

It appears the Muxer needs a ufrag to identify the connections, which we also don't have in this context. 

So I'm not seeing a way to identify which connection was reset within the muxer

Reply:
To add to this: Eventually the reset connection is polled by the swarm behaviour and then closed with `SwarmEvent::ConnectionClosed`. So information on the connection can be obtained from the Swarm when this happens, if desired.